cmd/acra-connector/acra-connector.go

```diff
@@ -229,6 +229,19 @@ func main() {
 	tlsCert := flag.String("tls_cert", "", "Path to certificate")
 	tlsAcraserverSNI := flag.String("tls_acraserver_sni", "", "Expected Server Name (SNI) from AcraServer")
 	tlsAuthType := flag.Int("tls_auth", int(tls.RequireAndVerifyClientCert), "Set authentication mode that will be used in TLS connection with AcraServer/AcraTranslator. Values in range 0-4 that set auth type (https://golang.org/pkg/crypto/tls/#ClientAuthType). Default is tls.RequireAndVerifyClientCert")
+	tlsOcspURL := flag.String("tls_ocsp_url", "", "OCSP service URL")
+	tlsOcspRequired := flag.String("tls_ocsp_required", network.OcspRequiredDenyUnknownStr,
+		fmt.Sprintf("How to treat certificates unknown to OCSP: <%s>", strings.Join(network.OcspRequiredValuesList, "|")))
+	tlsOcspFromCert := flag.String("tls_ocsp_from_cert", network.OcspFromCertPreferStr,
+		fmt.Sprintf("How to treat OCSP server described in certificate itself: <%s>", strings.Join(network.OcspFromCertValuesList, "|")))
+	tlsOcspCheckOnlyLeafCertificate := flag.Bool("tls_ocsp_check_only_leaf_certificate", false, "Put 'true' to check only final/last certificate, or 'false' to check the whole certificate chain using OCSP")
+	tlsCrlURL := flag.String("tls_crl_url", "", "URL of the Certificate Revocation List (CRL) to use")
+	tlsCrlFromCert := flag.String("tls_crl_from_cert", network.CrlFromCertPreferStr,
+		fmt.Sprintf("How to treat CRL URL described in certificate itself: <%s>", strings.Join(network.CrlFromCertValuesList, "|")))
+	tlsCrlCheckOnlyLeafCertificate := flag.Bool("tls_crl_check_only_leaf_certificate", false, "Put 'true' to check only final/last certificate, or 'false' to check the whole certificate chain using CRL")
+	tlsCrlCacheSize := flag.Uint("tls_crl_cache_size", 16, "How many CRLs to cache in memory (use 0 to disable caching)")
+	tlsCrlCacheTime := flag.Uint("tls_crl_cache_time", 0,
```

Comment:
if 0 means no expiration then better to use some named constant for that. That how we did it for keystore cache - https://github.com/cossacklabs/acra/blob/master/keystore/cache.go#L21

Reply:
Done. Created const `CrlDisableCacheTime = 0`, put as default value for `tls_crl_cache_time`.
Also, `CrlDefaultCacheSize = 16` as default value for `tls_crl_cache_size`